packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -894,12 +893,12 @@ class ThemeData extends Diagnosticable {
         primaryIconTheme,
         accentIconTheme,
         sliderTheme,
-        chipTheme,
+        tabBarTheme,
```

Comment:
Does the order actually matter here?

Reply:
For the sanity's sake, the properties appear in the same order in the class, the .raw constructor, etc.